src/components/instructor/SetDetails/HomeworkSet.vue

```diff
@@ -29,35 +29,42 @@
 </template>
 
 <script lang="ts">
-import { defineComponent, ref, watch, toRefs } from 'vue';
+import { defineComponent, ref, watch, computed } from 'vue';
 import { useQuasar } from 'quasar';
-import { cloneDeep } from 'lodash-es';
+import { useRoute } from 'vue-router';
+import { parseRouteSetID } from 'src/router/utils';
 
 import HomeworkDates from './HomeworkDates.vue';
-import { HomeworkSet } from 'src/store/models/problem_sets';
+import { HomeworkSet } from 'src/common/models/problem_sets';
 import { useStore } from 'src/store';
+import { cloneDeep } from 'lodash';
```

Comment:
😢

Reply:
Still working on this.  I thought that calling the clone method of a client-side model would replicate this, but it doesn't seem to.